register.lic

```diff
@@ -0,0 +1,76 @@
+=begin
+  Search a deed register for the specified crafting material.
+  Output all lines that match.
+=end
+
+custom_require.call(%w[common common-items drinfomon])
+
+class Register
+  include DRC
+  include DRCI
+
+  def initialize
+    settings = get_settings
```

Comment:
The performance hit is probably negligible, but you could `get_settings` after you parse args - so settings are never retrieved if args are invalid.

Reply:
Ok